instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -286,9 +278,13 @@ def process_request(self, request):
             request.META[self._environ_token] = token
 
         if _DjangoMiddleware._otel_request_hook:
-            _DjangoMiddleware._otel_request_hook(  # pylint: disable=not-callable
-                span, request
-            )
+            try:
+                _DjangoMiddleware._otel_request_hook(  # pylint: disable=not-callable
+                    span, request
+                )
+            except Exception as exception:
+                # process_response() will not be called, so we need to clean up
```

Comment:
are we recording the exception? Should it be asserted in the test?

Reply:
I think we settled on just logging the exception and not recording it to the trace:

> If you could remove the portions about recording unexpected behavior in the span used for the original request we can move forward with the pr. We can just handle the exception with a log.

Reply:
Indeed, thanks!